modules/ROOT/pages/clustering/monitoring/endpoints.adoc

```diff
@@ -114,8 +115,8 @@ true* Closing connection #0
 The  status endpoint, available at `/db/<databasename>/cluster/status`, is to be used to assist with rolling upgrades.
 For more information, see link:https://neo4j.com/docs/upgrade-migration-guide/current/version-5/upgrade-minor/#_clusters[Upgrade and Migration Guide -> Clusters].
 
-Typically, it is desired to have some guarantee that a Primary is safe to shutdown for each database before removing it from a cluster.
-Counter-intuitively, that a Primary is safe to shutdown means that a majority of the *other* Primaries are healthy, caught up, and have recently heard from that database's leader.
+Typically, it is desired to have some guarantee that a primary is safe to shutdown for each database before removing it from a cluster.
+Counter-intuitively, that a primary is safe to shutdown means that a majority of the *other* primaries are healthy, caught up, and have recently heard from that database's leader.
 The status endpoints provide the following information in order to help resolve such issues.
 
 [NOTE]
```

Comment:
line 124, <<clustering-transacting-via-raft-protocol, Raft>> - this cross-ref doesn't work. Do we have anything to redirect it to?

Reply:
No, we don't. We got rid of the unnecessary 'advanced' sections in the move to v5. So probably just get rid of the link.